frontend/apps/reader/modules/readermenu.lua

```diff
@@ -250,10 +250,14 @@ function ReaderMenu:setUpdateItemTable()
 
     if Device:supportsScreensaver() then
         local ss_book_settings = {
-            text = _("Exclude this book's cover from screensaver"),
+            text = _("Exclude this book's cover and text from screensaver"),
```

Comment:
What's this book's text?

Reply:
Like the actual words of the book

Reply:
Could you explain what that means? What will toggling this do?

Reply:
You know, the letters and sentences and stuff you read in the book. So if you have the book open on your screen, like on page 5. When you go to screensaver (set to "Leave as it is"), if you don't want this particular book's text to remain on the screen, this checkbox will show you a random image instead of keeping the words on the screen. Exact same thing as how the previous "Exclude this book's cover" behavior applies to "Use last book's cover", but expanded to also apply to "Leave as it is".

Reply:
So does this cover it?
```suggestion
            text = _("Use random image as screensaver for this book"),
```

Reply:
Hmmm I feel like that doesn't give the right "This is an exception" vibe as before

Reply:
Plus the user doesn't really care about what's shown instead, just that the book's details are not shown

Reply:
```suggestion
            text = _("Use random image as screensaver only for this book"),
```

Reply:
> Plus the user doesn't really care about what's shown instead […]  I think we should still have the string communicate that it's a privacy thing like before

The user cares what it actually does. What it doesn't do doesn't exactly help with that. If it's a privacy thing, the current wording doesn't convey that.

Reply:
It does convey it with "Exclude". That sends a message that this is a privacy exception made for this book. This wording I think would camouflage its purpose. And we still allow "Reading progress" as a screensaver type that's not excluded, so it's not just random image. These wordings wouldn't be accurate

Reply:
Great! We're finally communicating. ;-)

I always took exclude to mean "blergh this cover's ugly". Which could be as simple as this:
<img src=https://user-images.githubusercontent.com/202757/207315075-8d69cab4-151c-4a63-aa80-71e095ea0218.png width=30%>

Anyway, `Exclude this book's content and cover from screensaver` should work.

Reply:
Yeah I like that wording!

Reply:
I wasn't trying to be flippant or impatient either, I was just genuinely puzzled there on getting my meaning across :)